src/js/letters/containers/Main.jsx

```diff
@@ -10,9 +12,19 @@ class Main extends React.Component {
   }
 
   render() {
+    let appContent;
+
+    if (this.props.lettersAvailable) {
+      appContent = this.props.children;
+    } else if (this.props.lettersAvailable === undefined) {
```

Comment:
Ah yes good point, in that case perhaps the possible values should be `available`, `unavailable`, `awaitingResponse` or something like that? Although that will mean the checks for `this.props.available` everywhere will have to change and be a little less obvious. And yes agree about perhaps needing a different status if the app grows in complexity.